Gems/Atom/RHI/Vulkan/Code/Source/RHI/SwapChain.cpp

```diff
@@ -348,6 +348,63 @@ namespace AZ
             }
         }
 
+        void SwapChain::SetHDRMetaData(float maxOutputNits, float minOutputNits, float maxContentLightLevel,
+            float maxFrameAverageLightLevel) {
+            auto& device = static_cast<Device&>(GetDevice());
+
+            //From DX12 RHI
+            struct DisplayChromacities
+            {
+                float RedX;
+                float RedY;
+                float GreenX;
+                float GreenY;
+                float BlueX;
+                float BlueY;
+                float WhiteX;
+                float WhiteY;
+            };
+            static const DisplayChromacities DisplayChromacityList[] =
+            {
+                { 0.64000f, 0.33000f, 0.30000f, 0.60000f, 0.15000f, 0.06000f, 0.31270f, 0.32900f }, // Display Gamut Rec709
+                { 0.70800f, 0.29200f, 0.17000f, 0.79700f, 0.13100f, 0.04600f, 0.31270f, 0.32900f }, // Display Gamut Rec2020
+            };
+
+            // Select the chromaticity based on HDR format
+            int selectedChroma = 0;
+            if (m_colorSpace == VK_COLOR_SPACE_EXTENDED_SRGB_LINEAR_EXT)
+            {
+                selectedChroma = 0;
+            }
+            else if (m_colorSpace == VK_COLOR_SPACE_HDR10_ST2084_EXT)
+            {
+                selectedChroma = 1;
+            }
+            else
+            {
+                AZ_Assert(false, "Unhandled color space for swapchain.");
+            }
+
+            // const float LuminanceScalingFactor = 10000.0f;
+
+            const DisplayChromacities& Chroma = DisplayChromacityList[selectedChroma];
+            VkHdrMetadataEXT hdrMetadata;
+            hdrMetadata.sType = VK_STRUCTURE_TYPE_HDR_METADATA_EXT;
+            hdrMetadata.displayPrimaryRed = {Chroma.RedX, Chroma.RedY};
+            hdrMetadata.displayPrimaryGreen = {Chroma.GreenX, Chroma.GreenY};
+            hdrMetadata.displayPrimaryBlue = {Chroma.BlueX, Chroma.BlueY};
+            hdrMetadata.whitePoint = {Chroma.WhiteX, Chroma.WhiteY};
```

Comment:
These values seem to be scaled on DX12 side. Lets add a comment why we are not doing it here so it doesnt seem like a mistake.